### PR TITLE
fix: remove offset and add from to next uri

### PR DESCRIPTION
**Pull request by ArneD**

### CODE REVIEW

File: src/MunicipalityRegistry.Api.Legacy/Municipality/MunicipalityController.cs [hunk 1]
```diff
@@ -316,13 +324,23 @@ private static IEnumerable<GeografischeNaam> GetGemeentenamenByLanguage(Municipa
...
+                : null;
+        }
+
+        private static Uri BuildNextSyncUri(PaginationInfo paginationInfo, long? from, string nextUrlBase)
+        {
+            var offset = paginationInfo.Offset;
+            var limit = paginationInfo.Limit;
+
+            return from.HasValue
+                ? new Uri(string.Format(nextUrlBase, from.Value + 1, limit))
```
Comment (ridingwolf) [inline src/MunicipalityRegistry.Api.Legacy/Municipality/MunicipalityController.cs:343]: why **from + 1**, if i pass **from=100** I would expect to get position 100 too

Comment (CumpsD) [reply src/MunicipalityRegistry.Api.Legacy/Municipality/MunicipalityController.cs:343]: I think this is to build the next url. You ask from=100 and get 100-200, and the next url will be from=201?

Comment (CumpsD) [reply src/MunicipalityRegistry.Api.Legacy/Municipality/MunicipalityController.cs:343]: But if that is the case, it should probable be from + limit + 1

File: src/MunicipalityRegistry.Api.Legacy/appsettings.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
     "Id": "https://legacy.staging-basisregisters.vlaanderen/v1/feeds/gemeenten.atom",
     "Title": "Basisregisters Vlaanderen - Gemeenteregister",
     "Self": "https://legacy.staging-basisregisters.vlaanderen/syndication/feed/municipality.atom",
-    "NextUri": "https://legacy.staging-basisregisters.vlaanderen/v1/feeds/gemeenten.atom?offset={0}&limit={1}",
+    "NextUri": "https://legacy.staging-basisregisters.vlaanderen/v1/feeds/gemeenten.atom?from={0}&limit={1}",
```
Comment (CumpsD) [inline src/MunicipalityRegistry.Api.Legacy/appsettings.json:20]: Need to keep in mind to do this in aws-infrastructure repo too

File: src/MunicipalityRegistry.Api.Legacy/Municipality/MunicipalityController.cs [hunk 3]
```diff
@@ -199,8 +199,6 @@ await context
                 filtering.Filter?.ContainsObject ?? false)
                 .Fetch(filtering, sorting, pagination);
```
Comment (CumpsD) [inline src/MunicipalityRegistry.Api.Legacy/Municipality/MunicipalityController.cs:200]: While we're at it, can you indent this `.Fetch` please

File: src/MunicipalityRegistry.Api.Legacy/Municipality/MunicipalityController.cs (hunk 1 above)
Comment (ArneD) [reply src/MunicipalityRegistry.Api.Legacy/Municipality/MunicipalityController.cs:343]: `from` is inclusive. So yes, `from = 100` will give you event 100.
However the naming does not reflect the intent.

**Pull request merged**